### PR TITLE
LPS-202874 Client extension zip does not get deployed from a project with a version declared

### PR DESCRIPTION
**Pull request by drewbrokke**
This is an update for [LPS-202874](https://issues.liferay.com/browse/LPS-202874).

### CODE REVIEW

File: modules/sdk/gradle-plugins-workspace/src/gradleTest/clientExtensionDeploy/build.gradle [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import com.liferay.gradle.util.OSDetector
+
+task runGradleTest
+
+runGradleTest {
+	dependsOn ":with-version:deploy", ":without-version:deploy"
```
Comment (petershin) [inline modules/sdk/gradle-plugins-workspace/src/gradleTest/clientExtensionDeploy/build.gradle:6]: for the `clientExtensionXXX` tests, most of them follow this pattern: `xyz-client-extension`. lets apply that here too for consistency

File: modules/sdk/gradle-plugins-workspace/src/gradleTest/clientExtensionDeploy/build.gradle [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+	doLast {
+		FileTree fileTree = project.fileTree("bundles/osgi/client-extensions")
+
+		assert fileTree.files.size() == 2
```
Comment (petershin) [inline modules/sdk/gradle-plugins-workspace/src/gradleTest/clientExtensionDeploy/build.gradle:11]: avoid chaining:
```
Set<File> files = fileTree.files
```

**Review by petershin — CHANGES_REQUESTED** [review]

File: modules/sdk/gradle-plugins-workspace/src/gradleTest/clientExtensionDeploy/build.gradle (hunk 1 above)
Comment (drewbrokke) [reply modules/sdk/gradle-plugins-workspace/src/gradleTest/clientExtensionDeploy/build.gradle]: Done

File: modules/sdk/gradle-plugins-workspace/src/gradleTest/clientExtensionDeploy/build.gradle (hunk 2 above)
Comment (drewbrokke) [reply modules/sdk/gradle-plugins-workspace/src/gradleTest/clientExtensionDeploy/build.gradle]: Done

**Comment by drewbrokke** [comment]
Just pushed the recommended changes.

**Comment by petershin** [comment]
ci:test:sf

**Comment by petershin** [comment]
ci:forward

**Review by petershin — APPROVED** [review]

**Comment by drewbrokke** [comment]
Closing because this is not the correct fix according to the Slack thread. The zip should never have a version in it.

cc @gamerson

**Comment by drewbrokke** [comment]
See https://github.com/drewbrokke/liferay-portal/pull/1573

**Pull request closed without merging**